src/lib/project.ts

```diff
@@ -273,29 +273,29 @@ export class Project implements ProjectInfo {
   async getCommit(commit: string): Promise<Commit> {
     return new Request(
       `projects/${this.id}/commits/${commit}`,
-      this.seed.api,
+      this.seed.seed,
```

Comment:
thoughts on this `this.seed.seed` usage?

Reply:
Seems weird, without remembering the details I'd say it should be `this.seed.host`